templates/config.json

```
@@ -537,6 +578,15 @@
       "provider": "",
       "summary": "The organization incorporates best practices such as least-privilege or zero-trust in its security operating model.",
       "guidance": "This describes the high level operating principles of a DevOps oriented security program. The cloud-native, zero-trust approach enables a modern security operating model and eliminates the need for certain legacy security controls.",
+      "instructions": [
+        "Are you a zero-trust network?",
```

"zero-trust" network is not a thing.

```suggestion
        "Do rely on traditional network perimeters or leverage a zero-trust architecture?",
```